src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin itemSize($item-size) {
+  width: $item-size;
+  height: $item-size;
+}
+
+$days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
```

Comment:
"mon" 0,
  "tue" (101px),
  "wed" (101px * 2),
  "thu" (101px * 3),
  "fri" (101px * 4),
  "sat" (101px * 5),
  "sun" (101px * 6);

try to use this code, it will add a first day of the month on a top of your calendar 
https://prnt.sc/L7vK229-xq9J